src/analyze/analyze-security.c

```diff
@@ -124,7 +124,8 @@ struct security_assessor {
                 char **ret_description);
         size_t offset;
         uint64_t parameter;
-        bool default_dependencies_only;
+        bool default_dependencies_only:1;
+        bool system_services_only:1;
```

Comment:
I know Lennart said to do this, but please drop ":1". Because of alignment requirements this doesn't save any space, but makes the code to handle the struct longer.